packages/cudajit/cudajit.0.5.0/opam

```diff
@@ -0,0 +1,53 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Bindings to the `cuda` and `nvrtc` libraries with a unified interface"
+description:
+  "Bindings to manually selected parts of `lcuda` and `lnvrtc`, with a few types and conversion functions to facilitate use."
+maintainer: ["Lukasz Stafiniak"]
```

Comment:
Could you add a contact email here? (This is part of a tighter policy we are trying to ease in.)

Reply:
Done (forgot about this).